### DISCUSSION TITLE
Cannot delete Blob storage because exceeded Outgoing Bandwidth

### DISCUSSION THREAD

**Discussion by ricardofiorani** [vercel/community#5300] [Feature Question] [answered]
### Question

1. I created a blob storage to try it out. 
2. The tiny 100mb of Outgoing Bandwidth was exceeded.
3. I decided to remove the blob storage because it doesn't fulfill my use case needs
4. Go to the "delete" page -> "You must remove all the blobs before deleting this store."
5. Go to manage blob "browser" page -> "An error has occurred while fetching your blobs."
6. "You have reached your usage limits for this store using the Hobby plan. — Upgrade to Pro to increase the limits."

Now I have things stored in there that I cannot delete unless I upgrade my plan.

Are my files really kidnapped unless I pay a ransom, or did I miss some hidden button that flushes it all?

**Comment by gempesaw** [comment]
howdy vercel folks, I would also appreciate a resolution to this problem. I've run into the exact same issue: outgoing bandwidth exceeded, decided I don't want to use vercel blob, I can't delete my Blob because it has files in it; can't delete the files cuz the UI is disabled. 

please advise! cheers

**Reply by Munemh** [reply]
Found any solutions?

**Comment by alukach** [comment]
I believe this is akin to https://github.com/orgs/vercel/discussions/4437, wherein you are indeed locked-out until you either upgrade your plan or your limits reset.

**Comment by amyegan** [comment, marked as answer]
Hi everyone! We made a change that should let you view the list and delete blobs even after limits are exceeded. Please give it a try and let me know whether or not it works for you

**Comment by vvo** [comment]
@ricardofiorani Also we just upgrade the Hobby limit to 5GB for Vercel Blob 👍
